pipeline.md

```diff
@@ -57,8 +45,5 @@ Log back into the Development Cluster, and create the production secret, passing
 oc login <dev cluster>
 oc process -f deploy/prod-credentials.yml -p API_URL=${API_URL} REGISTRY_URL=${REGISTRY_HOSTNAME} TOKEN=${TOKEN} | oc apply -f- -n field-guides-dev
 ```
-## Deploy pipeline
 
-```
-oc process -f build/field-guides-build.yml --param-file=build/dev/params | oc apply -f-
-```
+At this point the pipeline should be fully functional and able to build, deploy, and promote the site.
```

Comment:
Executing the pipeline fails because the `s2i-gows` imagestream was created in the my current project `myproject` instead of the `field-guides-dev` project.  Error message from pipeline below
```
The ImageStreamTag "s2i-gows:latest" is invalid: from: Error resolving ImageStreamTag s2i-gows:latest in namespace field-guides-dev: imagestreams.image.openshift.io "s2i-gows" not found
```